### PR TITLE
Issue/2933 post preview

### LINKED ISSUE #2933: Calypsofication: Redesign ViewPostFragment
The post preview screen which currently appears after tapping a post in the post list needs a redesign.
* Drop the toolbar below the preview (including the ability to comment)
* Add an "edit" button to the toolbar to open the post for editing
* Use Merriweather font for post title and content
* Drop the use of a TextView for local drafts (should be able to use the WebView that's already used for uploaded posts)

Note: #2875 should be completed and merged before tackling this.

### PR DESCRIPTION
**Pull request by nbradbury**
Resolves #2933 - replaces `ViewPostFragment` with a new `PostPreviewFragment` that better suits the new Calypso posts list. To see it in action, simply tap a post in the post list.

Before & after screens below:

before-and-after

### CODE REVIEW

File: WordPress/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -239,6 +239,8 @@
     <string name="posts_fetching">Fetching posts…</string>
     <string name="pages_fetching">Fetching pages…</string>
     <string name="toast_err_post_uploading">Unable to open post while it\'s uploading</string>
+    <string name="post_preview_title">Preview Post</string>
```
Comment (tonyr59h) [inline WordPress/src/main/res/values/strings.xml]: These don't appear to be used, there are other strings, preview_post/page, that are being used as the title.

**Comment by tonyr59h** [comment]
Should we have landscape layout fill the width of the screen? It looks like it fills the same width as portrait currently.

**Comment by nbradbury** [comment]
> Should we have landscape layout fill the width of the screen?

Many of our screens use larger left/right margins in landscape (and even larger on tablets). You'll see this in posts, pages, reader, and stats, and I think it works quite well.

For comparison, here's a Nexus 10 showing the post preview. The top screen is with the current margins, the bottom is without them. I think the extra whitespace is preferable to longer lines of text to read.

margins

File: WordPress/src/main/res/values/strings.xml (hunk 1 above)
Comment (nbradbury) [inline WordPress/src/main/res/values/strings.xml]: Fixed in 0759f71

**Comment by daniloercoli** [comment]
> I think the extra whitespace is preferable to longer lines of text to read.

+:100:

**Comment by tonyr59h** [comment]
:+1: :shipit:

**Pull request merged**